docs/examples/example_lime.ipynb

```diff
@@ -74,18 +74,19 @@
     "* showcase the features with highest global importance\n",
     "\n",
     "In line with this ambition, they define a notion of 'coverage' -- to\n",
```

Comment:
coverage of what? What should be covered?

Reply:
This is defined in the LIME paper and they call it coverage as the idea is to maximize it by choosing a subset of the samples, which are the ones shown to the user, and try to make its explanations "cover" the majority of the important features.

Reply:
Ahhh, I see. I was thinking more of "coverage" in the statistical sense... Ok so how about "they define a notion of 'coverage' specifying how well candidate datapoints in V are explained by features that are relevant overall for many observed datapoints. The goal is to find V that is not larger than some pre-specified size such that this coverage is maximal."

Reply:
Ah I understand, it can be tricky, I added your comment:
[173c2a2](https://github.com/Quantco/metalearners/pull/21/commits/173c2a27083c4f06df717009069d57a3e0afb11a)